cmd/config.go

```diff
@@ -320,6 +320,12 @@ func (c *Config) getDefaultData() (map[string]interface{}, error) {
 		return nil, err
 	}
 
+	kernelInfo, err := getKernelInfo(c.fs)
+	if err == nil {
+		data["kernel"] = kernelInfo
```

Comment:
Please only set `data["kernel"]` if `kernelInfo != nil`.

Reply:
I guess I could also just return file not found from the no-op implementation

Reply:
Return a file not found error would have the same effect, but I think returning `nil, nil` more clearly signals the intent of the code.

Reply:
This isn't quite right: if `err == nil` and `kernelInfo == nil` then the code ends up at line 327.

Reply:
Yeah, I was a bit too quick with that fix